src/js/angular/ttyg/directives/chat-view.directive.js

```diff
@@ -0,0 +1,179 @@
+import {ChatQuestion} from "../../models/ttyg/chat-question";
+import {TTYGEventName} from "../services/ttyg-context.service";
+import {chatQuestionToChatMessageMapper} from "../services/chat-message.mapper";
+import {cloneDeep} from "lodash";
+
+const modules = [];
+
+angular
+    .module('graphdb.framework.ttyg.directives.chat-view', modules)
+    .directive('chatView', ChartViewComponent);
+
+ChartViewComponent.$inject = ['toastr', '$translate', 'TTYGContextService', 'TTYGService'];
+
+/**
+ * @ngdoc directive
+ * @name graphdb.framework.ttyg.directives.chat-view:chatView
+ * @restrict E
+ * @description
+ *
+ * This directive represents a chat view component that allows users to interact with a chat. It provides functionality for asking questions,
+ * regenerating chat content, and other interactive features.
+ *
+ * It encapsulates the logic and presentation for displaying and managing chat-related interactions.
+ *
+ * @example
+ * <chat-view></chat-view>
+ */
+function ChartViewComponent(toastr, $translate, TTYGContextService, TTYGService) {
```

Comment:
This should be ChatPanelComponent. Where this ChatView came from?

Reply:
Renamed